breakable_toy/.gitignore

```diff
@@ -0,0 +1 @@
+.directory
```

Comment:
Having a pattern in the `.gitignore` at the root of the project should be sufficient to get a it ignored throughout the project.